packages/shared/src/components/squads/Comment.tsx

```diff
@@ -159,24 +191,9 @@ export function SquadComment({
               onInput={onInputChange}
               hint={linkHint}
               saveHintSpace
+              disabled={isCheckingUrl}
```

Comment:
what if it fails to load?

Reply:
This won't be disabled then. The `isLoading` is handled properly inside react-query whether the request succeeded or not.